src/styles/style.css

```diff
@@ -0,0 +1,66 @@
+@font-face {
+  font-family: Avenir-Book;
+  src: url(/src/fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(/src/fonts/Avenir-Heavy.ttf);
+}
+
+.page__body {
+  margin-top: 0;
+}
+
+.body__search {
+  width: 100%;
+  margin-top: 20px;
+  display: block;
+  color: #3d4e61;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  font-family: Avenir-Book, sans-serif;
+
+  background-image: url(../images/Search.svg);
+  background-repeat: no-repeat;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+}
+
+.body__search--big {
+  background-size: 19px;
+  height: 70px;
+  padding-left: 62px;
+  background-position: left;
+  background-position-x: 26px;
+  font-size: 16px;
+}
+
+.body__search--small {
+  font-size: 14px;
+  height: 42px;
+  padding-left: 33px;
+  background-position: left;
+  background-position-x: 13px;
+  background-size: 11px;
+}
+
+.body__search--big::placeholder {
+  color: #3d4e61;
+  font-weight: 300;
+  line-height: 19px;
+}
+
+.body__search--small::placeholder {
+  color: #3d4e61;
+  font-weight: 400;
```

Comment:
There is an inconsistency in the font-weight of the placeholder text between '.body__search--big' and '.body__search--small'. The 'big' version uses font-weight 300, while the 'small' version uses font-weight 400. Consider making them consistent unless there is a specific reason for this difference.